modules/cnf-cpu-infra-container.adoc

```diff
@@ -5,17 +5,62 @@
 [id="cnf-cpu-infra-container_{context}"]
 = Restricting CPUs for infra and application containers
 
-You can reserve cores (threads) from a single NUMA node for operating system housekeeping tasks and put your workloads on another NUMA node. Partitioning the CPUs this way can prevent the housekeeping processes from impacting latency-sensitive application processes. By default, CRI-O uses all online CPUs to run infra containers on nodes, which can result in context switches and spikes in latency. 
+Housekeeping and workloads may use the central processing units (CPUs) in a way that would impact latency sensitive processes running on those CPUs. Partitioning the CPUs and assigning the processes prevents the processes from interfering with each other. By default, CRI-O uses all online CPUs to run infra containers on nodes, which can result in context switches and spikes in latency. Use the following table to assign processes:
 
-You can ensure that housekeeping tasks and workloads run on separate NUMA nodes by specifying two groups of CPUs in the `spec` section of the performance profile. 
+.Process assignments
+[%header,cols=3*] 
+|===
+|Core	
+|Process
+|Details
+
+|0
+|Interrupts
+|Redirected to reserved CPUs (optional in 4.7+)
+
+|1
+|Kernel processes
+|Pinned to reserved CPUs
+
+|2
+|OS processes/systemd services
+|Pinned to reserved CPUs
+
+|3
+|Infrastructure pods
+|Runs on any CPU not used by a low latency container, including reserved
+
+|4
+|General workflow pod
+|Runs on any CPU not used by a low latency container, including reserved
+
+|5
+|Latency sensitive workload pods
+|Pinned to a specific set of exclusive CPUs from the isolated pool
+|===
+
+The exact partitioning pattern to use depends on many factors like hardware, workload characteristics and the expected system load. Some sample use cases are as follows:
+
+* If the latency sensitive workload uses specific hardware (like network interface cards (NICs)), ensure that the CPUs in the isolated pool are as "close" to this hardware as possible. Place the workload in at least the same Non-Uniform Memory Access (NUMA) node.
+
+* The reserved pool is used for handling all interrupts. When depending on system networking, allocate a sufficiently sized reserve pool to handle all the incoming packet interrupts. In version 4.7+, workloads can optionally be labeled as sensitive. More information regarding the management of device interrupt processing can be found in link:https://docs.openshift.com/container-platform/4.8/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#managing-device-interrupt-processing-for-guaranteed-pod-isolated-cpus_cnf-master[Managing device interrupt processing for guaranteed pod isolated cpus].
```

Comment:
You should probably make the link relative only or point to the same version of docs using some macro like {version} (?)

Reply:
Ah yes, a macro would make the most sense! Thanks for pointing that out.

Reply:
Updated to "In {product-version} and later" and updated link to reflect xref and .adoc:

> xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#managing-device-interrupt-processing-for-guaranteed-pod-isolated-cpus[Managing device interrupt processing for guaranteed pod isolated CPUs]